ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -504,9 +504,8 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
       if (rp.label === 'Other') {
         children = <TextArea value={this.state.workspace.researchPurpose.otherDisseminateResearchFindings}
                              onChange={v => this.updateOtherDisseminateResearch(v)}
-                             placeholder='Specify the name of the forum (journal, scientific
-                             conference, blog etc.) through which you will disseminate your
-                             findings, if available.'
+                             placeholder={'Specify the name of the forum (journal, scientific conference, blog etc.)' +
```

Comment:
There was weird spacing here, because it was including all the whitespace at the start of each line. This fixes that.
<img width="441" alt="Screen Shot 2020-04-22 at 11 32 25 AM" src="https://user-images.githubusercontent.com/15351021/80001973-05283e00-848d-11ea-8266-60e92f9bfcc7.png">